Framework/PythonInterface/mantid/kernel/src/Exports/InstrumentInfo.cpp

```diff
@@ -46,9 +64,38 @@ void export_InstrumentInfo() {
            return_value_policy<copy_const_reference>(),
            "Returns the facility that contains this instrument.")
 
+      .def("liveListener", &InstrumentInfo::liveListener,
+           liveListener_overloads(
+               args("self", "name"),
+               "Returns the name of the specific LiveListener class that is "
+               "used "
+               "by the given connection name. If no name is provided, the "
+               "default connection is used."))
+
+      // Unclear why this is named "instdae", leaving in case legacy req'd
       .def("instdae", &InstrumentInfo::liveDataAddress, arg("self"),
            "Returns the host name and the port of the machine hosting DAE and "
            "providing port to connect to for a live data stream")
 
+      .def("liveDataAddress", &InstrumentInfo::liveDataAddress,
+           liveDataAddress_overloads(
+               args("self", "name"),
+               "Returns the Address string of a live data connection on this "
+               "instrument. If no connection name is provided, the default "
+               "connection is used."))
+
+      .def("liveListenerInfo", &InstrumentInfo::liveListenerInfo,
+           return_value_policy<copy_const_reference>(),
+           liveListenerInfo_overloads(
+               args("self", "name"), "Returns a LiveListenerInfo instance for"))
+
+      .def("hasLiveListenerInfo", &InstrumentInfo::hasLiveListenerInfo,
+           arg("self"),
+           "Returns true if this instrument has at least one LiveListenerInfo")
+
+      .def("liveListenerInfoList", &InstrumentInfo::liveListenerInfoList,
+           return_value_policy<copy_const_reference>(), arg("self"),
+           "Returns all available LiveListenerInfo instances as a vector")
+
```

Comment:
No updates seem to have been made to the [tests](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/test/python/mantid/kernel)